[2025-04-28T10:52:39.440+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_data_pipeline.submit_traffic_processor_spark_job manual__2025-04-26T07:15:01.084538+00:00 [queued]>
[2025-04-28T10:52:39.450+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_data_pipeline.submit_traffic_processor_spark_job manual__2025-04-26T07:15:01.084538+00:00 [queued]>
[2025-04-28T10:52:39.451+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 1
[2025-04-28T10:52:39.467+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): submit_traffic_processor_spark_job> on 2025-04-26 07:15:01.084538+00:00
[2025-04-28T10:52:39.473+0000] {standard_task_runner.py:60} INFO - Started process 216 to run task
[2025-04-28T10:52:39.479+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'traffic_data_pipeline', 'submit_traffic_processor_spark_job', 'manual__2025-04-26T07:15:01.084538+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/traffic_pipeline_dag.py', '--cfg-path', '/tmp/tmpdq9ltkd0']
[2025-04-28T10:52:39.482+0000] {standard_task_runner.py:88} INFO - Job 84: Subtask submit_traffic_processor_spark_job
[2025-04-28T10:52:39.591+0000] {task_command.py:423} INFO - Running <TaskInstance: traffic_data_pipeline.submit_traffic_processor_spark_job manual__2025-04-26T07:15:01.084538+00:00 [running]> on host 02173e944d7b
[2025-04-28T10:52:39.714+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_data_pipeline' AIRFLOW_CTX_TASK_ID='submit_traffic_processor_spark_job' AIRFLOW_CTX_EXECUTION_DATE='2025-04-26T07:15:01.084538+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-26T07:15:01.084538+00:00'
[2025-04-28T10:52:39.715+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-28T10:52:39.717+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n            echo "--- Checking Line Endings for processor.py ---"\n            cat -e /opt/spark/app/processor.py | head -n 10\n            echo "--- Finished Checking Line Endings ---"\n            echo "Setting PYTHONPATH..."\n            PYTHON_USER_SITE="/home/***/.local/lib/python3.10/site-packages"\n            export PYTHONPATH="${PYTHON_USER_SITE}:${PYTHONPATH:-}"\n            echo "Using PYTHONPATH=$PYTHONPATH"\n            echo "Exporting other environment variables..."\n            export KAFKA_BROKERS_INTERNAL="$KAFKA_BROKERS_INTERNAL" && \\\n            export KAFKA_TOPIC="$KAFKA_TOPIC" && \\\n            export SCHEMA_REGISTRY_URL="$SCHEMA_REGISTRY_URL" && \\\n            export REDIS_HOST="$REDIS_HOST" && \\\n            export CLICKHOUSE_HOST="$CLICKHOUSE_HOST" && \\\n            export CLICKHOUSE_NATIVE_PORT="$CLICKHOUSE_NATIVE_PORT" && \\\n            export MINIO_ENDPOINT="$MINIO_ENDPOINT" && \\\n            export MINIO_ACCESS_KEY="$MINIO_ACCESS_KEY" && \\\n            export MINIO_SECRET_KEY="$MINIO_SECRET_KEY" && \\\n            export MINIO_BUCKET="$MINIO_BUCKET" && \\\n            CHECKPOINT_LOCATION_VAR="s3a://${MINIO_BUCKET:-traffic-data}/checkpoints_***/traffic_processing_pipeline" && \\\n            export CHECKPOINT_LOCATION="$CHECKPOINT_LOCATION_VAR" && \\\n            export PYSPARK_PYTHON=/usr/local/bin/python3.10 && \\\n            echo "--- Running Spark Submit ---" && \\\n            spark-submit \\\n            --master local[*] \\\n            --deploy-mode client \\\n            --conf spark.sql.streaming.schemaInference=true \\\n            --conf spark.hadoop.fs.s3a.endpoint="$MINIO_ENDPOINT" \\\n            --conf spark.hadoop.fs.s3a.access.key="$MINIO_ACCESS_KEY" \\\n            --conf spark.hadoop.fs.s3a.secret.key="$MINIO_SECRET_KEY" \\\n            --conf spark.hadoop.fs.s3a.path.style.access=true \\\n            --conf spark.hadoop.fs.s3a.impl=org.apache.hadoop.fs.s3a.S3AFileSystem \\\n            --conf spark.sql.avro.schemaRegistryUrl="$SCHEMA_REGISTRY_URL" \\\n            /opt/spark/app/processor.py\n            ']
[2025-04-28T10:52:39.739+0000] {subprocess.py:86} INFO - Output:
[2025-04-28T10:52:39.740+0000] {subprocess.py:93} INFO - --- Checking Line Endings for processor.py ---
[2025-04-28T10:52:39.745+0000] {subprocess.py:93} INFO - import os$
[2025-04-28T10:52:39.746+0000] {subprocess.py:93} INFO - import json$
[2025-04-28T10:52:39.747+0000] {subprocess.py:93} INFO - import math$
[2025-04-28T10:52:39.747+0000] {subprocess.py:93} INFO - from datetime import datetime$
[2025-04-28T10:52:39.748+0000] {subprocess.py:93} INFO - import traceback$
[2025-04-28T10:52:39.749+0000] {subprocess.py:93} INFO - from typing import Optional$
[2025-04-28T10:52:39.749+0000] {subprocess.py:93} INFO - $
[2025-04-28T10:52:39.750+0000] {subprocess.py:93} INFO - # Import thM-FM-0 viM-aM-;M-^Gn cM-aM-:M-'n thiM-aM-:M-?t cho Pandas UDF$
[2025-04-28T10:52:39.751+0000] {subprocess.py:93} INFO - import pandas as pd$
[2025-04-28T10:52:39.751+0000] {subprocess.py:93} INFO - from pyspark.sql import SparkSession$
[2025-04-28T10:52:39.752+0000] {subprocess.py:93} INFO - --- Finished Checking Line Endings ---
[2025-04-28T10:52:39.752+0000] {subprocess.py:93} INFO - Setting PYTHONPATH...
[2025-04-28T10:52:39.753+0000] {subprocess.py:93} INFO - Using PYTHONPATH=/home/***/.local/lib/python3.10/site-packages:
[2025-04-28T10:52:39.754+0000] {subprocess.py:93} INFO - Exporting other environment variables...
[2025-04-28T10:52:39.754+0000] {subprocess.py:93} INFO - --- Running Spark Submit ---
[2025-04-28T10:52:39.759+0000] {subprocess.py:93} INFO - /opt/spark/bin/load-spark-env.sh: line 68: ps: command not found
[2025-04-28T10:52:45.311+0000] {subprocess.py:93} INFO - --- PySpark Script Dependencies Imported ---
[2025-04-28T10:52:45.312+0000] {subprocess.py:93} INFO - --- Starting Spark Streaming Traffic Processor ---
[2025-04-28T10:52:45.313+0000] {subprocess.py:93} INFO - Reading from Kafka: kafk-1:9092,kafk-2:9092,kafk-3:9092 / Topic: raw_traffic_data
[2025-04-28T10:52:45.313+0000] {subprocess.py:93} INFO - Avro Schema File: /opt/spark/schemas/raw_traffic_event.avsc
[2025-04-28T10:52:45.314+0000] {subprocess.py:93} INFO - Writing state to Redis: redis:6379
[2025-04-28T10:52:45.315+0000] {subprocess.py:93} INFO - Writing history to ClickHouse: clickhouse-server:8123 (DB: traffic_db)
[2025-04-28T10:52:45.315+0000] {subprocess.py:93} INFO - Writing Parquet to MinIO Bucket: s3a://traffic-data/
[2025-04-28T10:52:45.316+0000] {subprocess.py:93} INFO - Checkpoint Location Base: s3a://traffic-data/checkpoints_***/traffic_processing_pipeline
[2025-04-28T10:52:45.481+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 INFO SparkContext: Running Spark version 3.3.4
[2025-04-28T10:52:45.640+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-04-28T10:52:45.876+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 INFO ResourceUtils: ==============================================================
[2025-04-28T10:52:45.877+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-04-28T10:52:45.879+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 INFO ResourceUtils: ==============================================================
[2025-04-28T10:52:45.880+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 INFO SparkContext: Submitted application: TrafficProcessorDE
[2025-04-28T10:52:45.963+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-04-28T10:52:45.974+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 INFO ResourceProfile: Limiting resource is cpu
[2025-04-28T10:52:45.975+0000] {subprocess.py:93} INFO - 25/04/28 10:52:45 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-04-28T10:52:46.093+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SecurityManager: Changing view acls to: ***
[2025-04-28T10:52:46.094+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SecurityManager: Changing modify acls to: ***
[2025-04-28T10:52:46.095+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SecurityManager: Changing view acls groups to:
[2025-04-28T10:52:46.096+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SecurityManager: Changing modify acls groups to:
[2025-04-28T10:52:46.098+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2025-04-28T10:52:46.491+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO Utils: Successfully started service 'sparkDriver' on port 39125.
[2025-04-28T10:52:46.574+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SparkEnv: Registering MapOutputTracker
[2025-04-28T10:52:46.642+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SparkEnv: Registering BlockManagerMaster
[2025-04-28T10:52:46.687+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-04-28T10:52:46.688+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-04-28T10:52:46.693+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-04-28T10:52:46.733+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-bc2317ec-2ac5-4db9-a42e-31fe70fadb6d
[2025-04-28T10:52:46.758+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-04-28T10:52:46.782+0000] {subprocess.py:93} INFO - 25/04/28 10:52:46 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-04-28T10:52:47.168+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2025-04-28T10:52:47.187+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2025-04-28T10:52:47.448+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO Executor: Starting executor ID driver on host 02173e944d7b
[2025-04-28T10:52:47.458+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2025-04-28T10:52:47.499+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 36063.
[2025-04-28T10:52:47.500+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO NettyBlockTransferService: Server created on 02173e944d7b:36063
[2025-04-28T10:52:47.501+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2025-04-28T10:52:47.509+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 02173e944d7b, 36063, None)
[2025-04-28T10:52:47.516+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO BlockManagerMasterEndpoint: Registering block manager 02173e944d7b:36063 with 434.4 MiB RAM, BlockManagerId(driver, 02173e944d7b, 36063, None)
[2025-04-28T10:52:47.517+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 02173e944d7b, 36063, None)
[2025-04-28T10:52:47.522+0000] {subprocess.py:93} INFO - 25/04/28 10:52:47 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 02173e944d7b, 36063, None)
[2025-04-28T10:52:48.633+0000] {subprocess.py:93} INFO - Spark Session created. Spark version: 3.3.4
[2025-04-28T10:52:48.635+0000] {subprocess.py:93} INFO - Using PySpark version: 3.3.4
[2025-04-28T10:52:48.635+0000] {subprocess.py:93} INFO - Broadcasting segments data...
[2025-04-28T10:52:48.931+0000] {subprocess.py:93} INFO - Segments data broadcasted successfully.
[2025-04-28T10:52:49.456+0000] {subprocess.py:93} INFO - /opt/spark/python/lib/pyspark.zip/pyspark/sql/pandas/functions.py:394: UserWarning: In Python 3.6+ and Spark 3.0+, it is preferred to specify type hints for pandas UDF instead of specifying pandas UDF type which will be deprecated in the future releases. See SPARK-28264 for more details.
[2025-04-28T10:52:49.457+0000] {subprocess.py:93} INFO - Pandas UDF for map matching defined.
[2025-04-28T10:52:49.458+0000] {subprocess.py:93} INFO - Setting up Kafka stream reader...
[2025-04-28T10:52:49.458+0000] {subprocess.py:93} INFO - Kafka Read Options: {'kafka.bootstrap.servers': 'kafk-1:9092,kafk-2:9092,kafk-3:9092', 'subscribe': 'raw_traffic_data', 'startingOffsets': 'earliest', 'failOnDataLoss': 'false'}
[2025-04-28T10:52:54.247+0000] {subprocess.py:93} INFO - Kafka stream reader loaded.
[2025-04-28T10:52:54.249+0000] {subprocess.py:93} INFO - Reading Avro schema from file: /opt/spark/schemas/raw_traffic_event.avsc
[2025-04-28T10:52:54.252+0000] {subprocess.py:93} INFO - Successfully read Avro schema string from file.
[2025-04-28T10:52:54.253+0000] {subprocess.py:93} INFO - Setting up Avro deserialization...
[2025-04-28T10:52:54.644+0000] {subprocess.py:93} INFO - Avro deserialization step defined.
[2025-04-28T10:52:54.645+0000] {subprocess.py:93} INFO - Defining data processing steps...
[2025-04-28T10:52:55.294+0000] {subprocess.py:93} INFO - Data processing steps defined.
[2025-04-28T10:52:55.295+0000] {subprocess.py:93} INFO - Using checkpoint location: s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2
[2025-04-28T10:52:55.296+0000] {subprocess.py:93} INFO - Starting streaming query...
[2025-04-28T10:52:55.658+0000] {subprocess.py:93} INFO - 25/04/28 10:52:55 WARN MetricsConfig: Cannot locate configuration: tried hadoop-metrics2-s3a-file-system.properties,hadoop-metrics2.properties
[2025-04-28T10:52:57.427+0000] {subprocess.py:93} INFO - 25/04/28 10:52:57 WARN ResolveWriteToStream: spark.sql.adaptive.enabled is not supported in streaming DataFrames/Datasets and will be disabled.
[2025-04-28T10:52:57.889+0000] {subprocess.py:93} INFO - Streaming query 'traffic_processing_pipeline_debug2' started successfully.
[2025-04-28T10:53:05.365+0000] {subprocess.py:93} INFO - 25/04/28 10:53:05 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:05.367+0000] {subprocess.py:93} INFO - 25/04/28 10:53:05 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:05.368+0000] {subprocess.py:93} INFO - 25/04/28 10:53:05 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.708+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.710+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.711+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.712+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.715+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.715+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.718+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.721+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.724+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.724+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.725+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.726+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.727+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.728+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.728+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.729+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.730+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.731+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.732+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.733+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.734+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.738+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.740+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.742+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.743+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.746+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.750+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.751+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.753+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.754+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.755+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.756+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.756+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.757+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.758+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.758+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.759+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.760+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.760+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.761+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.762+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.763+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.764+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.765+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.766+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.766+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.767+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.768+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.768+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.769+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.770+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.770+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.771+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.772+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.772+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.773+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.774+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.774+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.775+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.776+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.776+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.777+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.778+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.778+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.779+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.780+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.780+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.781+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.782+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.783+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.783+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.784+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.784+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.785+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.786+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.786+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.787+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.788+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.788+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.789+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.790+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.791+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.792+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.794+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.796+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.798+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.798+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.799+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.800+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.800+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.801+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.801+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.802+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.803+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.803+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.804+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.804+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.805+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.806+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.807+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.808+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.809+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.810+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.811+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.812+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.814+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.816+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.817+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.819+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.820+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.822+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.822+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.823+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.824+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.825+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.826+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.827+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.828+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.828+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.829+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.830+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.831+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.831+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.832+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.833+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.834+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.834+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.835+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.836+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.839+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.842+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.843+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.844+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.844+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.845+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.846+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.847+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.847+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.848+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.849+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.849+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.850+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.851+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.852+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.855+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.856+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.857+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.857+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.858+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.859+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.862+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.865+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.869+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.873+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.874+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.874+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.875+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.876+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.877+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.877+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.878+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.879+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.880+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.881+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.882+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.882+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.883+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.884+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.885+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.886+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.887+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.890+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.892+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.894+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.895+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.897+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.897+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.898+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.901+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.901+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.902+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.903+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.905+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.906+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.907+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.907+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.908+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.909+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.909+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.910+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.910+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.914+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.916+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.917+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.918+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.919+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.920+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.921+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.922+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.923+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.924+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.924+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.925+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.926+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.927+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.927+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.929+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.930+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.932+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.934+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.935+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.936+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.937+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.938+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.939+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.940+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.941+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.941+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.942+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.943+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.944+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.945+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.946+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.946+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.947+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.948+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.950+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.952+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.954+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.955+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.956+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.957+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.958+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.959+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.960+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.963+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.965+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.966+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.966+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.967+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.968+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.969+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.969+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.970+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.971+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.971+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.972+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.973+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.974+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.974+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.975+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.976+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.977+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:07.977+0000] {subprocess.py:93} INFO - 25/04/28 10:53:07 WARN KafkaDataConsumer: KafkaDataConsumer is not running in UninterruptibleThread. It may hang when KafkaDataConsumer's methods are interrupted because of KAFKA-1894
[2025-04-28T10:53:12.300+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.330+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.361+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.424+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.484+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.557+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.600+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.637+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.705+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.825+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.844+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:12.992+0000] {subprocess.py:93} INFO - 25/04/28 10:53:12 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:13.037+0000] {subprocess.py:93} INFO - 25/04/28 10:53:13 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:13.082+0000] {subprocess.py:93} INFO - 25/04/28 10:53:13 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:13.104+0000] {subprocess.py:93} INFO - 25/04/28 10:53:13 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:13.152+0000] {subprocess.py:93} INFO - 25/04/28 10:53:13 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:15.948+0000] {subprocess.py:93} INFO - 25/04/28 10:53:15 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:16.163+0000] {subprocess.py:93} INFO - 25/04/28 10:53:16 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:16.236+0000] {subprocess.py:93} INFO - 25/04/28 10:53:16 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:16.331+0000] {subprocess.py:93} INFO - 25/04/28 10:53:16 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:16.453+0000] {subprocess.py:93} INFO - 25/04/28 10:53:16 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:16.567+0000] {subprocess.py:93} INFO - 25/04/28 10:53:16 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:18.154+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:18.331+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:18.455+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:18.560+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:18.612+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:18.762+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:18.772+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN CheckpointFileManager: Failed to rename temp file s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/16/.9.delta.a22f4b69-1cdb-4df8-af5f-2b87c5b3d924.TID19.tmp to s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/16/9.delta because file exists
[2025-04-28T10:53:18.832+0000] {subprocess.py:93} INFO - org.apache.hadoop.fs.FileAlreadyExistsException: Failed to rename s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/16/.9.delta.a22f4b69-1cdb-4df8-af5f-2b87c5b3d924.TID19.tmp to s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/16/9.delta; destination file exists
[2025-04-28T10:53:18.842+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1920)
[2025-04-28T10:53:18.844+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2025-04-28T10:53:18.849+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2025-04-28T10:53:18.851+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2025-04-28T10:53:18.853+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2025-04-28T10:53:18.856+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2025-04-28T10:53:18.858+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2025-04-28T10:53:18.862+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileSystem.rename(FileSystem.java:1624)
[2025-04-28T10:53:18.869+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.DelegateToFileSystem.renameInternal(DelegateToFileSystem.java:206)
[2025-04-28T10:53:18.876+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.AbstractFileSystem.renameInternal(AbstractFileSystem.java:790)
[2025-04-28T10:53:18.892+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.AbstractFileSystem.rename(AbstractFileSystem.java:720)
[2025-04-28T10:53:18.894+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileContext.rename(FileContext.java:1036)
[2025-04-28T10:53:18.897+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.FileContextBasedCheckpointFileManager.renameTempFile(CheckpointFileManager.scala:346)
[2025-04-28T10:53:18.899+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.CheckpointFileManager$RenameBasedFSDataOutputStream.close(CheckpointFileManager.scala:154)
[2025-04-28T10:53:18.900+0000] {subprocess.py:93} INFO - 	at net.jpountz.lz4.LZ4BlockOutputStream.close(LZ4BlockOutputStream.java:198)
[2025-04-28T10:53:18.903+0000] {subprocess.py:93} INFO - 	at java.base/java.io.FilterOutputStream.close(FilterOutputStream.java:188)
[2025-04-28T10:53:18.906+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider.finalizeDeltaFile(HDFSBackedStateStoreProvider.scala:450)
[2025-04-28T10:53:18.907+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider.org$apache$spark$sql$execution$streaming$state$HDFSBackedStateStoreProvider$$commitUpdates(HDFSBackedStateStoreProvider.scala:320)
[2025-04-28T10:53:18.917+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider$HDFSBackedStateStore.commit(HDFSBackedStateStoreProvider.scala:141)
[2025-04-28T10:53:18.918+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.StreamingAggregationStateManagerBaseImpl.commit(StreamingAggregationStateManager.scala:89)
[2025-04-28T10:53:18.924+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec$$anon$2.$anonfun$close$3(statefulOperators.scala:481)
[2025-04-28T10:53:18.927+0000] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2025-04-28T10:53:18.931+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:642)
[2025-04-28T10:53:18.943+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreWriter.timeTakenMs(statefulOperators.scala:142)
[2025-04-28T10:53:18.951+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreWriter.timeTakenMs$(statefulOperators.scala:142)
[2025-04-28T10:53:18.952+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec.timeTakenMs(statefulOperators.scala:352)
[2025-04-28T10:53:18.953+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec$$anon$2.close(statefulOperators.scala:481)
[2025-04-28T10:53:18.954+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.NextIterator.closeIfNeeded(NextIterator.scala:66)
[2025-04-28T10:53:18.955+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.NextIterator.hasNext(NextIterator.scala:75)
[2025-04-28T10:53:18.955+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.hashAgg_doAggregateWithKeys_0$(Unknown Source)
[2025-04-28T10:53:18.962+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-04-28T10:53:18.964+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-04-28T10:53:18.966+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenExec$$anon$1.hasNext(WholeStageCodegenExec.scala:760)
[2025-04-28T10:53:18.968+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:18.969+0000] {subprocess.py:93} INFO - 	at org.apache.spark.ContextAwareIterator.hasNext(ContextAwareIterator.scala:39)
[2025-04-28T10:53:18.972+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:18.979+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:18.983+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$GroupedIterator.fill(Iterator.scala:1211)
[2025-04-28T10:53:19.001+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$GroupedIterator.hasNext(Iterator.scala:1217)
[2025-04-28T10:53:19.005+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:19.010+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator.foreach(Iterator.scala:943)
[2025-04-28T10:53:19.018+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator.foreach$(Iterator.scala:943)
[2025-04-28T10:53:19.019+0000] {subprocess.py:93} INFO - 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
[2025-04-28T10:53:19.030+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.PythonRDD$.writeIteratorToStream(PythonRDD.scala:307)
[2025-04-28T10:53:19.031+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.python.PythonUDFRunner$$anon$1.writeIteratorToStream(PythonUDFRunner.scala:53)
[2025-04-28T10:53:19.044+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.BasePythonRunner$WriterThread.$anonfun$run$1(PythonRunner.scala:438)
[2025-04-28T10:53:19.047+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.logUncaughtExceptions(Utils.scala:2066)
[2025-04-28T10:53:19.049+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.BasePythonRunner$WriterThread.run(PythonRunner.scala:272)
[2025-04-28T10:53:19.050+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN CheckpointFileManager: Failed to rename temp file s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/18/.9.delta.aba0454d-3953-40eb-a099-7bb18ffd3123.TID21.tmp to s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/18/9.delta because file exists
[2025-04-28T10:53:19.052+0000] {subprocess.py:93} INFO - org.apache.hadoop.fs.FileAlreadyExistsException: Failed to rename s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/18/.9.delta.aba0454d-3953-40eb-a099-7bb18ffd3123.TID21.tmp to s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/18/9.delta; destination file exists
[2025-04-28T10:53:19.054+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1920)
[2025-04-28T10:53:19.055+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2025-04-28T10:53:19.059+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2025-04-28T10:53:19.063+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2025-04-28T10:53:19.063+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2025-04-28T10:53:19.064+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2025-04-28T10:53:19.065+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2025-04-28T10:53:19.069+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileSystem.rename(FileSystem.java:1624)
[2025-04-28T10:53:19.070+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.DelegateToFileSystem.renameInternal(DelegateToFileSystem.java:206)
[2025-04-28T10:53:19.072+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.AbstractFileSystem.renameInternal(AbstractFileSystem.java:790)
[2025-04-28T10:53:19.075+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.AbstractFileSystem.rename(AbstractFileSystem.java:720)
[2025-04-28T10:53:19.076+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileContext.rename(FileContext.java:1036)
[2025-04-28T10:53:19.077+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.FileContextBasedCheckpointFileManager.renameTempFile(CheckpointFileManager.scala:346)
[2025-04-28T10:53:19.078+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.CheckpointFileManager$RenameBasedFSDataOutputStream.close(CheckpointFileManager.scala:154)
[2025-04-28T10:53:19.079+0000] {subprocess.py:93} INFO - 	at net.jpountz.lz4.LZ4BlockOutputStream.close(LZ4BlockOutputStream.java:198)
[2025-04-28T10:53:19.079+0000] {subprocess.py:93} INFO - 	at java.base/java.io.FilterOutputStream.close(FilterOutputStream.java:188)
[2025-04-28T10:53:19.081+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider.finalizeDeltaFile(HDFSBackedStateStoreProvider.scala:450)
[2025-04-28T10:53:19.088+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider.org$apache$spark$sql$execution$streaming$state$HDFSBackedStateStoreProvider$$commitUpdates(HDFSBackedStateStoreProvider.scala:320)
[2025-04-28T10:53:19.090+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider$HDFSBackedStateStore.commit(HDFSBackedStateStoreProvider.scala:141)
[2025-04-28T10:53:19.093+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.StreamingAggregationStateManagerBaseImpl.commit(StreamingAggregationStateManager.scala:89)
[2025-04-28T10:53:19.098+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec$$anon$2.$anonfun$close$3(statefulOperators.scala:481)
[2025-04-28T10:53:19.102+0000] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2025-04-28T10:53:19.108+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:642)
[2025-04-28T10:53:19.118+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreWriter.timeTakenMs(statefulOperators.scala:142)
[2025-04-28T10:53:19.120+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreWriter.timeTakenMs$(statefulOperators.scala:142)
[2025-04-28T10:53:19.122+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec.timeTakenMs(statefulOperators.scala:352)
[2025-04-28T10:53:19.126+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec$$anon$2.close(statefulOperators.scala:481)
[2025-04-28T10:53:19.132+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.NextIterator.closeIfNeeded(NextIterator.scala:66)
[2025-04-28T10:53:19.133+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.NextIterator.hasNext(NextIterator.scala:75)
[2025-04-28T10:53:19.134+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.hashAgg_doAggregateWithKeys_0$(Unknown Source)
[2025-04-28T10:53:19.136+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-04-28T10:53:19.139+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-04-28T10:53:19.140+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenExec$$anon$1.hasNext(WholeStageCodegenExec.scala:760)
[2025-04-28T10:53:19.149+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:19.151+0000] {subprocess.py:93} INFO - 	at org.apache.spark.ContextAwareIterator.hasNext(ContextAwareIterator.scala:39)
[2025-04-28T10:53:19.152+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:19.155+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:19.160+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$GroupedIterator.fill(Iterator.scala:1211)
[2025-04-28T10:53:19.162+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$GroupedIterator.hasNext(Iterator.scala:1217)
[2025-04-28T10:53:19.166+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:19.167+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator.foreach(Iterator.scala:943)
[2025-04-28T10:53:19.172+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator.foreach$(Iterator.scala:943)
[2025-04-28T10:53:19.178+0000] {subprocess.py:93} INFO - 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
[2025-04-28T10:53:19.181+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.PythonRDD$.writeIteratorToStream(PythonRDD.scala:307)
[2025-04-28T10:53:19.182+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.python.PythonUDFRunner$$anon$1.writeIteratorToStream(PythonUDFRunner.scala:53)
[2025-04-28T10:53:19.183+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.BasePythonRunner$WriterThread.$anonfun$run$1(PythonRunner.scala:438)
[2025-04-28T10:53:19.185+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.logUncaughtExceptions(Utils.scala:2066)
[2025-04-28T10:53:19.187+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.BasePythonRunner$WriterThread.run(PythonRunner.scala:272)
[2025-04-28T10:53:19.188+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.190+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.191+0000] {subprocess.py:93} INFO - 25/04/28 10:53:18 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.195+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.197+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.258+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.295+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.342+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.467+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.629+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.718+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.802+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:19.830+0000] {subprocess.py:93} INFO - 25/04/28 10:53:19 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:20.290+0000] {subprocess.py:93} INFO - 25/04/28 10:53:20 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:20.401+0000] {subprocess.py:93} INFO - 25/04/28 10:53:20 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:20.508+0000] {subprocess.py:93} INFO - 25/04/28 10:53:20 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:20.639+0000] {subprocess.py:93} INFO - 25/04/28 10:53:20 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:20.711+0000] {subprocess.py:93} INFO - 25/04/28 10:53:20 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:20.768+0000] {subprocess.py:93} INFO - 25/04/28 10:53:20 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:20.908+0000] {subprocess.py:93} INFO - 25/04/28 10:53:20 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:20.982+0000] {subprocess.py:93} INFO - 25/04/28 10:53:20 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:21.053+0000] {subprocess.py:93} INFO - 25/04/28 10:53:21 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:21.137+0000] {subprocess.py:93} INFO - 25/04/28 10:53:21 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:21.226+0000] {subprocess.py:93} INFO - 25/04/28 10:53:21 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:21.266+0000] {subprocess.py:93} INFO - 25/04/28 10:53:21 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:21.344+0000] {subprocess.py:93} INFO - 25/04/28 10:53:21 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:21.454+0000] {subprocess.py:93} INFO - 25/04/28 10:53:21 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:21.643+0000] {subprocess.py:93} INFO - 25/04/28 10:53:21 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:21.959+0000] {subprocess.py:93} INFO - 25/04/28 10:53:21 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:22.248+0000] {subprocess.py:93} INFO - 25/04/28 10:53:22 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:22.817+0000] {subprocess.py:93} INFO - 25/04/28 10:53:22 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:24.848+0000] {subprocess.py:93} INFO - 25/04/28 10:53:24 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:25.115+0000] {subprocess.py:93} INFO - 25/04/28 10:53:25 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:25.314+0000] {subprocess.py:93} INFO - 25/04/28 10:53:25 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:25.619+0000] {subprocess.py:93} INFO - 25/04/28 10:53:25 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:25.830+0000] {subprocess.py:93} INFO - 25/04/28 10:53:25 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:25.960+0000] {subprocess.py:93} INFO - 25/04/28 10:53:25 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.101+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.155+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.206+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.282+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.379+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.439+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.534+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.584+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.630+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.651+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:26.976+0000] {subprocess.py:93} INFO - 25/04/28 10:53:26 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:27.242+0000] {subprocess.py:93} INFO - 25/04/28 10:53:27 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:53:27.683+0000] {subprocess.py:93} INFO - 25/04/28 10:53:27 WARN CheckpointFileManager: Failed to rename temp file s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/58/.9.delta.af56254b-0817-4c1d-9776-c2542f9cc556.TID61.tmp to s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/58/9.delta because file exists
[2025-04-28T10:53:27.759+0000] {subprocess.py:93} INFO - org.apache.hadoop.fs.FileAlreadyExistsException: rename destination s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/58/9.delta already exists.
[2025-04-28T10:53:27.782+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileSystem.rename(FileSystem.java:1600)
[2025-04-28T10:53:27.794+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.DelegateToFileSystem.renameInternal(DelegateToFileSystem.java:206)
[2025-04-28T10:53:27.797+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.AbstractFileSystem.renameInternal(AbstractFileSystem.java:790)
[2025-04-28T10:53:27.800+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.AbstractFileSystem.rename(AbstractFileSystem.java:720)
[2025-04-28T10:53:27.801+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileContext.rename(FileContext.java:1036)
[2025-04-28T10:53:27.802+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.FileContextBasedCheckpointFileManager.renameTempFile(CheckpointFileManager.scala:346)
[2025-04-28T10:53:27.803+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.CheckpointFileManager$RenameBasedFSDataOutputStream.close(CheckpointFileManager.scala:154)
[2025-04-28T10:53:27.804+0000] {subprocess.py:93} INFO - 	at net.jpountz.lz4.LZ4BlockOutputStream.close(LZ4BlockOutputStream.java:198)
[2025-04-28T10:53:27.896+0000] {subprocess.py:93} INFO - 	at java.base/java.io.FilterOutputStream.close(FilterOutputStream.java:188)
[2025-04-28T10:53:27.898+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider.finalizeDeltaFile(HDFSBackedStateStoreProvider.scala:450)
[2025-04-28T10:53:27.899+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider.org$apache$spark$sql$execution$streaming$state$HDFSBackedStateStoreProvider$$commitUpdates(HDFSBackedStateStoreProvider.scala:320)
[2025-04-28T10:53:27.900+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider$HDFSBackedStateStore.commit(HDFSBackedStateStoreProvider.scala:141)
[2025-04-28T10:53:27.900+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.StreamingAggregationStateManagerBaseImpl.commit(StreamingAggregationStateManager.scala:89)
[2025-04-28T10:53:27.901+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec$$anon$2.$anonfun$close$3(statefulOperators.scala:481)
[2025-04-28T10:53:27.902+0000] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2025-04-28T10:53:27.903+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:642)
[2025-04-28T10:53:27.903+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreWriter.timeTakenMs(statefulOperators.scala:142)
[2025-04-28T10:53:27.904+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreWriter.timeTakenMs$(statefulOperators.scala:142)
[2025-04-28T10:53:27.905+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec.timeTakenMs(statefulOperators.scala:352)
[2025-04-28T10:53:27.906+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec$$anon$2.close(statefulOperators.scala:481)
[2025-04-28T10:53:27.907+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.NextIterator.closeIfNeeded(NextIterator.scala:66)
[2025-04-28T10:53:27.908+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.NextIterator.hasNext(NextIterator.scala:75)
[2025-04-28T10:53:27.909+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.hashAgg_doAggregateWithKeys_0$(Unknown Source)
[2025-04-28T10:53:27.909+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-04-28T10:53:27.910+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-04-28T10:53:27.911+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenExec$$anon$1.hasNext(WholeStageCodegenExec.scala:760)
[2025-04-28T10:53:27.912+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:27.913+0000] {subprocess.py:93} INFO - 	at org.apache.spark.ContextAwareIterator.hasNext(ContextAwareIterator.scala:39)
[2025-04-28T10:53:27.914+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:27.915+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:27.915+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$GroupedIterator.fill(Iterator.scala:1211)
[2025-04-28T10:53:27.916+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$GroupedIterator.hasNext(Iterator.scala:1217)
[2025-04-28T10:53:27.917+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:53:27.918+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator.foreach(Iterator.scala:943)
[2025-04-28T10:53:27.919+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator.foreach$(Iterator.scala:943)
[2025-04-28T10:53:27.919+0000] {subprocess.py:93} INFO - 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
[2025-04-28T10:53:27.920+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.PythonRDD$.writeIteratorToStream(PythonRDD.scala:307)
[2025-04-28T10:53:27.921+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.python.PythonUDFRunner$$anon$1.writeIteratorToStream(PythonUDFRunner.scala:53)
[2025-04-28T10:53:27.921+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.BasePythonRunner$WriterThread.$anonfun$run$1(PythonRunner.scala:438)
[2025-04-28T10:53:27.922+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.logUncaughtExceptions(Utils.scala:2066)
[2025-04-28T10:53:27.923+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.BasePythonRunner$WriterThread.run(PythonRunner.scala:272)
[2025-04-28T10:53:30.717+0000] {job.py:213} ERROR - Job heartbeat got an exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.10/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "postgres" to address: Temporary failure in name resolution


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/jobs/job.py", line 187, in heartbeat
    self._merge_from(Job._fetch_from_db(self, session))
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/api_internal/internal_api_call.py", line 114, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/jobs/job.py", line 308, in _fetch_from_db
    session.merge(job)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 3056, in merge
    return self._merge(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 3136, in _merge
    merged = self.get(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 2853, in get
    return self._get_impl(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 2975, in _get_impl
    return db_load_fn(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/loading.py", line 530, in load_on_pk_identity
    session.execute(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1716, in execute
    conn = self._connection_for_bind(bind)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1555, in _connection_for_bind
    return self._transaction._connection_for_bind(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 750, in _connection_for_bind
    conn = bind.connect()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/future/engine.py", line 412, in connect
    return super(Engine, self).connect()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3325, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3404, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3374, in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2208, in _handle_dbapi_exception_noconnection
    util.raise_(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.10/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2025-04-28T10:53:31.455+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-04-28T10:53:48.183+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-04-28T10:54:06.024+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-04-28T10:54:25.929+0000] {subprocess.py:93} INFO - 25/04/28 10:54:25 WARN CheckpointFileManager: Failed to rename temp file s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/70/.9.delta.31cb3539-107e-4d53-8a9b-76d103711a66.TID73.tmp to s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/70/9.delta because file exists
[2025-04-28T10:54:26.024+0000] {subprocess.py:93} INFO - org.apache.hadoop.fs.FileAlreadyExistsException: Failed to rename s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/70/.9.delta.31cb3539-107e-4d53-8a9b-76d103711a66.TID73.tmp to s3a://traffic-data/checkpoints_***/traffic_processing_pipeline/traffic_processing_pipeline_debug2/state/0/70/9.delta; destination file exists
[2025-04-28T10:54:26.037+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1920)
[2025-04-28T10:54:26.054+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2025-04-28T10:54:26.058+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2025-04-28T10:54:26.060+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2025-04-28T10:54:26.062+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2025-04-28T10:54:26.064+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2025-04-28T10:54:26.065+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2025-04-28T10:54:26.066+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileSystem.rename(FileSystem.java:1624)
[2025-04-28T10:54:26.068+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.DelegateToFileSystem.renameInternal(DelegateToFileSystem.java:206)
[2025-04-28T10:54:26.069+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.AbstractFileSystem.renameInternal(AbstractFileSystem.java:790)
[2025-04-28T10:54:26.077+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.AbstractFileSystem.rename(AbstractFileSystem.java:720)
[2025-04-28T10:54:26.079+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileContext.rename(FileContext.java:1036)
[2025-04-28T10:54:26.080+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.FileContextBasedCheckpointFileManager.renameTempFile(CheckpointFileManager.scala:346)
[2025-04-28T10:54:26.081+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.CheckpointFileManager$RenameBasedFSDataOutputStream.close(CheckpointFileManager.scala:154)
[2025-04-28T10:54:26.082+0000] {subprocess.py:93} INFO - 	at net.jpountz.lz4.LZ4BlockOutputStream.close(LZ4BlockOutputStream.java:198)
[2025-04-28T10:54:26.084+0000] {subprocess.py:93} INFO - 	at java.base/java.io.FilterOutputStream.close(FilterOutputStream.java:188)
[2025-04-28T10:54:26.086+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider.finalizeDeltaFile(HDFSBackedStateStoreProvider.scala:450)
[2025-04-28T10:54:26.087+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider.org$apache$spark$sql$execution$streaming$state$HDFSBackedStateStoreProvider$$commitUpdates(HDFSBackedStateStoreProvider.scala:320)
[2025-04-28T10:54:26.089+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider$HDFSBackedStateStore.commit(HDFSBackedStateStoreProvider.scala:141)
[2025-04-28T10:54:26.090+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.state.StreamingAggregationStateManagerBaseImpl.commit(StreamingAggregationStateManager.scala:89)
[2025-04-28T10:54:26.091+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec$$anon$2.$anonfun$close$3(statefulOperators.scala:481)
[2025-04-28T10:54:26.093+0000] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2025-04-28T10:54:26.094+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:642)
[2025-04-28T10:54:26.096+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreWriter.timeTakenMs(statefulOperators.scala:142)
[2025-04-28T10:54:26.097+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreWriter.timeTakenMs$(statefulOperators.scala:142)
[2025-04-28T10:54:26.097+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec.timeTakenMs(statefulOperators.scala:352)
[2025-04-28T10:54:26.098+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.StateStoreSaveExec$$anon$2.close(statefulOperators.scala:481)
[2025-04-28T10:54:26.099+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.NextIterator.closeIfNeeded(NextIterator.scala:66)
[2025-04-28T10:54:26.100+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.NextIterator.hasNext(NextIterator.scala:75)
[2025-04-28T10:54:26.101+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.hashAgg_doAggregateWithKeys_0$(Unknown Source)
[2025-04-28T10:54:26.102+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-04-28T10:54:26.103+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-04-28T10:54:26.104+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenExec$$anon$1.hasNext(WholeStageCodegenExec.scala:760)
[2025-04-28T10:54:26.105+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:54:26.107+0000] {subprocess.py:93} INFO - 	at org.apache.spark.ContextAwareIterator.hasNext(ContextAwareIterator.scala:39)
[2025-04-28T10:54:26.108+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:54:26.110+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:54:26.112+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$GroupedIterator.fill(Iterator.scala:1211)
[2025-04-28T10:54:26.113+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$GroupedIterator.hasNext(Iterator.scala:1217)
[2025-04-28T10:54:26.113+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-04-28T10:54:26.114+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator.foreach(Iterator.scala:943)
[2025-04-28T10:54:26.115+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator.foreach$(Iterator.scala:943)
[2025-04-28T10:54:26.116+0000] {subprocess.py:93} INFO - 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
[2025-04-28T10:54:26.117+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.PythonRDD$.writeIteratorToStream(PythonRDD.scala:307)
[2025-04-28T10:54:26.118+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.python.PythonUDFRunner$$anon$1.writeIteratorToStream(PythonUDFRunner.scala:53)
[2025-04-28T10:54:26.119+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.BasePythonRunner$WriterThread.$anonfun$run$1(PythonRunner.scala:438)
[2025-04-28T10:54:26.119+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.logUncaughtExceptions(Utils.scala:2066)
[2025-04-28T10:54:26.120+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.python.BasePythonRunner$WriterThread.run(PythonRunner.scala:272)
[2025-04-28T10:54:26.387+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-04-28T10:54:42.597+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-04-28T10:54:58.683+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-04-28T10:55:16.686+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-04-28T10:55:38.032+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-04-28T10:56:02.458+0000] {subprocess.py:93} INFO - 25/04/28 10:56:02 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:56:02.540+0000] {subprocess.py:93} INFO - 25/04/28 10:56:02 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
[2025-04-28T10:56:02.631+0000] {subprocess.py:93} INFO - 25/04/28 10:56:02 WARN HDFSBackedStateStoreProvider: The state for version 8 doesn't exist in loadedMaps. Reading snapshot file and delta files if needed...Note that this is normal for the first batch of starting query.
